controllers/publishController.js

```diff
@@ -67,7 +67,7 @@ module.exports = {
         resolve(publishResults); // resolve the promise with the result from lbryApi.publishClaim;
       })
       .catch(error => {
-        logger.error('publishController.publish, error', error);
+        logger.error('caught a publishController.publish error');
```

Comment:
Shouldn't the actual error be logged here?

If not, ignore.

Reply:
The full error is passed back up the promise chain and then logged.  This particular log is more just for debugging, to be able to quickly ID if this error was triggered.